src/details/events.tsx

```diff
@@ -115,6 +125,67 @@ function eventPlace(entry: GedcomEntry) {
   return place?.data ? getData(place) : undefined;
 }
 
+function eventImages(entry: GedcomEntry, gedcom: GedcomData): Image[] {
+  return entry.tree
+    .filter((subEntry) => 'OBJE' === subEntry.tag)
+    .map((objectEntry) =>
+      dereference(objectEntry, gedcom, (gedcom) => gedcom.other),
+    )
+    .map((objectEntry) => getImageFileEntry(objectEntry))
+    .flatMap((imageFileEntry) =>
+      imageFileEntry
+        ? [
+            {
+              url: imageFileEntry?.data || '',
+              filename: getFileName(imageFileEntry) || '',
+            },
+          ]
+        : [],
+    );
+}
+
+function eventSources(entry: GedcomEntry, gedcom: GedcomData): Source[] {
+  return entry.tree
+    .filter((subEntry) => 'SOUR' === subEntry.tag)
+    .map((sourceEntryReference) => {
+      const sourceEntry = dereference(
+        sourceEntryReference,
+        gedcom,
+        (gedcom) => gedcom.other,
+      );
+
+      const title = sourceEntry.tree.find(
+        (subEntry) => 'TITL' === subEntry.tag,
+      );
```

Comment:
In my GEDCOM, I also have an ABBR tag under SOUR which behaves like a title.
My suggestion:
```
      const abbr = sourceEntry.tree.find(
        (subEntry) => 'ABBR' === subEntry.tag,
      );
(...)
      title: title?.data || abbr?.data,
```

Reply:
Good point, Thank you